src/services/db-manager.ts

```diff
@@ -57,6 +57,21 @@ export const baseConnConfig: PostgresConnectionOptions = {
   extra: { ssl: ['postgresql', 'localhost'].includes(config.dbHost) ? false : { rejectUnauthorized: false } },  // TODO: remove once DB instance with custom ssl cert is in place
 };
 
+export async function startDbLink(dbId: string, dbUser:string, dbPassword: string) {
+  const conn = await createConnection({
+    ...baseConnConfig,
+    name: `${dbId}-${Math.floor(Math.random()*10000)}-dblink`,
+    database: dbId,
+  });
+  console.log(dbUser, config.dbUser);
+  // create a dblink server to reuse the connections
+  // https://aws.amazon.com/blogs/database/migrating-oracle-autonomous-transactions-to-postgresql/
+  await conn.query(`CREATE EXTENSION IF NOT EXISTS dblink;`);
+  await conn.query(`CREATE SERVER IF NOT EXISTS loopback_dblink FOREIGN DATA WRAPPER dblink_fdw OPTIONS (host '${config.dbHost}', dbname '${dbId}', port '${config.dbPort}');`);
```

Comment:
What if you did this instead?

```suggestion
  await conn.query(`CREATE SERVER IF NOT EXISTS loopback_dblink FOREIGN DATA WRAPPER dblink_fdw OPTIONS (host 'localhost', dbname '${dbId}', port '${config.dbPort}');`);
```

It is the database itself talking to itself, so why go through the network card? There may also be special behavior granted to this form?

Reply:
tried it, but during testing it is already set to `localhost` I believe